cypress/e2e/formr-b/FormRB.spec.ts

```diff
@@ -188,18 +188,11 @@ describe("Form R (Part B) - desktop", () => {
     cy.get("[data-cy=LinkToNextSection] > .nhsuk-pagination__page").click();
 
     // -------- COVID Section ------------------------------------------------
+    //should be disabled whilst covide declaration flag is disabled in trainee-forms
     if (isCovid) {
```

Comment:
@EdwardBarclay Not too sure what's going on here (been a long day!). Looking at line 78 in this test file:  isCovid = true . Shouldn't isCovid be false (it is originally set to false on line 6) and then the if/else statements etc. can be left as they were before the fixes? Happy to look at it properly tomorrow.